our-work/software/aframe-volume-renderer.yml

```
@@ -0,0 +1,18 @@
+title: Web Volume Viewer
+description: Volumetric data visualization on web enabled for VR devices
+  browser technologies. It supports color mapping using transfer functions to map specific values of data to a scalar color. Developed using React-JS and AFrame.  
+languages:
+  - javascript
+  - reactjs
+tags:
+  - VR
+  - Volume Rendering
+  - WebGL
+people:
+  - name: Camilo Diaz Rodriguez
+    github_user: kmilo9999
+  - name: Robert Gemma
+    github_user: RobertGemmaJr
+links:
+  - category: code
+    url: https://github.com/brown-ccv/react-aframe-volume-renderer
```

Maybe add a link to A-Frame as well? It's not our code so I suppose it may not be beneficial and would be added to several different files

```suggestion
links:
  - category: code
    url: https://github.com/brown-ccv/react-volume-viewer.git
  - category: code
    url: https://www.npmjs.com/package/react-volume-viewer
```